tools/afl/generate/smessage_encrypt_decrypt.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019 Cossack Labs Limited
```

Comment:
```suggestion
 * Copyright (c) 2021 Cossack Labs Limited
```
(in other files too)

Reply:
that's not an issue, i believe

Reply:
In fact, those files *were* written in February 2019 primarily. It just took them some time to reach the repo 😂